integration_test/tests/wallet.rs

```diff
@@ -647,6 +659,20 @@ fn wallet__send__modelled() {
     model.unwrap();
 }
 
+#[cfg(not(feature = "v23_and_below"))]
+#[test]
+fn wallet__send_all__modelled() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    node.fund_wallet();
+    let address = node.client.new_address().expect("failed to create new address");
+
+    let recipients = vec![address.to_string()];
+
+    let json: SendAll = node.client.send_all(&recipients).expect("sendall");
```

Comment:
We can probably just use `&[address]` inline in `send_all`.

Reply:
Done